packages/shared/lib/services/connection.service.ts

```diff
@@ -945,18 +945,12 @@ class ConnectionService {
                     environment,
                     config
                 });
-
-                // if the credentials were refreshed be sure to set the last fetched date
-                await this.updateLastFetched(connection.id);
             }
 
             connection.credentials = response.credentials as OAuth2Credentials;
         }
 
-        // sample this to reduce writes and load on the db
-        if (Math.random() < 0.33) {
-            await this.updateLastFetched(connection.id);
-        }
```

Comment:
we were already doing it above.

Reply:
it was a tmp fix from khaliq, I think we can revert anyway as it was not directly the issue